components/Banner/Banner.styled.ts

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export const Background = styled.section`
+  width: 100%;
+  height: 40px;
+  background: #ebf4ee;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #75b587;
+  font-size: 14px;
+  font-family: GilroySemiBold;
+  cursor: pointer;
+
+  & .money {
```

Comment:
We should set up a styled-component instead of setting a `className` on the `span` with the money emoji.

Reply:
<img width="893" alt="Screen Shot 2021-03-23 at 11 23 42 AM" src="https://user-images.githubusercontent.com/26047842/112198408-4d367000-8bca-11eb-864d-0178502cbb2f.png">